targets/unity-v2/Testing/Tests/Client/ClientMultiUserTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿#if !DISABLE_PLAYFABCLIENT_API
+using System;
+using PlayFab.ClientModels;
+
+namespace PlayFab.UUnit
+{
+    public class ClientMultiUserTests : UUnitTestCase
+    {
+        private Action _tickAction;
+
+        private PlayFabAuthenticationContext _player1;
+        private PlayFabAuthenticationContext _player2;
+
+        public override void Tick(UUnitTestContext testContext)
+        {
+            if (_tickAction != null)
+                _tickAction();
+        }
+
+        private void SharedErrorCallback(PlayFabError error)
```

Comment:
I'm pretty much at the point where this method should be in PlayFabUUnitUtils beside ApiActionWrapper.
Every TestCase is independently copy-pasting this method.

Reply:
@Vicodex, can you make a tech debt task for us to do this someday?